main.go

```diff
@@ -43,6 +43,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
```

Comment:
I think we could and should support all available auth providers, so I'd rather you import the aggregator package
```suggestion
	_ "k8s.io/client-go/plugin/pkg/client/auth"
```